products/sle/main.pm

```diff
@@ -689,8 +700,8 @@ sub load_inst_tests {
         if (get_var('PATTERNS') || get_var('PACKAGES')) {
             loadtest "installation/installation_overview_before";
             loadtest "installation/select_patterns_and_packages";
-        }    # With SLE15 we change desktop using role and not by inselecting packages (Use SYSTEM_ROLE variable)
-        elsif (!check_var('DESKTOP', default_desktop) && !sle_version_at_least('15')) {
+        }  # With SLE15 we change desktop using role and not by inselecting packages (Use SYSTEM_ROLE variable), or if have minimalx, as there is no such a role
```

Comment:
s/inselecting/unselecting/, s/if have/if we have/.

Please break lines

Reply:
Fixed